.github/workflows/deploy-ss3-docs.yml

```diff
@@ -0,0 +1,158 @@
+          diff_label <- diff(label)
+          to_rm <- which(diff_label <10)
+          to_rm_lines <- label[to_rm+1]
+          for (i in to_rm_lines) {
+            start <- grep("<tr", html_txt, fixed = TRUE)
+            end <- grep("/tr", html_txt, fixed = TRUE)
+            start <- max(start[start %in% (i-5):(i-1)])
+            end <- min(end[end %in% (i+1):(i+5)])
+            if(length(start) == 1 & length(end) == 1) {
+              html_txt[start:end] <- ""
+            }
+          }
+          # remove LO HI INIT etc double header
+          label <- grep(">LO<", html_txt, fixed = TRUE)
+          diff_label <- diff(label)
+          to_rm <- which(diff_label < 20)
+          to_rm_lines <- label[to_rm+1]
+          for (i in to_rm_lines) {
+            start <- grep("<tr", html_txt, fixed = TRUE)
+            end <- grep("/tr", html_txt, fixed = TRUE)
+            start <- max(start[start %in% (i-10):(i-1)])
+            end <- min(end[end %in% (i+1):(i+10)])
+            if(length(start) == 1 & length(end) == 1) {
+              html_txt[start:end] <- ""
+            }
+          }
+          # remove Pattern N Parameters etc double header
+          label <- grep(">N Parameters<", html_txt, fixed = T)
+          diff_label <- diff(label)
+          to_rm <- which(diff_label == 5)
+          to_rm_lines <- label[to_rm+1]
+          for (i in to_rm_lines) {
+            start <- grep("<tr", html_txt, fixed = TRUE)
+            end <- grep("/tr", html_txt, fixed = TRUE)
+            start <- max(start[start %in% (i-2):(i-1)])
+            end <- min(end[end %in% (i+1):(i+2)])
+            if(length(start) == 1 & length(end) == 1) {
+              html_txt[start:end] <- ""
+            }
+          }
+          # Add title to table of contents
+          where_add <- grep('<nav id="TOC" role="doc-toc">',
+                            html_txt, fixed = TRUE)
+          html_txt <- append(html_txt,
+                             "<h1>Table of Contents</h1>",
+                             after = where_add[1])
+          # Add title to references
+          where_add <- (grep("references csl-bib-body hanging-indent",
+                             html_txt, fixed = TRUE) - 1)
+          html_txt <- append(html_txt,
+            '<h1 data-number="17" id="sec:references"><span class="header-section-number">17</span> References</h1>',
+            after = where_add)
+          # Add references to table of contents
+          where_add <- grep("</nav>", html_txt, fixed = TRUE)-2
+          html_txt <- append(html_txt,
+            '<li><a href="#sec:references"><span class="toc-section-number">17</span> References</a></li>',
+            after = where_add)
+          writeLines(html_txt, "SS330_User_Manual.html")
+        shell: Rscript {0}
+
+      - name: move html manual to the deployment folder
+        run: |
+          rm -f docs/SS330_User_Manual.html
+          mv SS330_User_Manual.html docs/SS330_User_Manual.html
```

Comment:
@kellijohnson-NOAA Since we are only deploying the release version of the manual to the website, I think that we can delete this portion. It created the "experimental" version that was on the website and now it moves a file to the deployment folder that is no longer in use. What are your thoughts?